src/Frontend/Pages/Auth/index.css

```diff
@@ -0,0 +1,205 @@
+#kash-modal {
+  display: flex;
+}
+.kash-modal {
+  top: 0;
+  display: flex;
+  flex-direction: row;
+  height: 65vh;
+  flex-flow: row wrap;
+  width: 40%;
+}
+.auth-left,
+.auth-right {
+  padding: 2rem 0.8rem 0.8rem 0.8rem;
+  display: flex;
+  flex-direction: column;
+}
+.auth-right {
+  padding-top: 2rem;
+}
+.auth-right {
+  width: 60%;
+  flex: 1;
+}
+
+.auth-left {
+  background-color: var(--primary);
+  color: var(--white);
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+  background-image: url("https://static-assets-web.flixcart.com/www/linchpin/fk-cp-zion/img/login_img_c4a81e.png");
+  background-position: center 85%;
+  background-repeat: no-repeat;
+  width: 40%;
+}
+.auth-left > h3,
+.auth-left-desc {
+  text-align: left;
+}
+.auth-left > .auth-left-desc {
+  padding: 0.5rem 0.5rem;
+  color: rgb(216, 216, 216);
+}
+.auth-left > img {
+  width: 80%;
+  position: absolute;
+  bottom: 1rem;
+}
+.auth-email {
+  position: relative;
+  width: 100%;
+  margin-bottom: 4vh;
+  display: flex;
+}
+.auth-email > input {
+  border: none;
+  outline: none;
+  border-bottom: 1px solid #e0e0e0;
+  width: 100%;
+}
+
+.auth-email > label {
+  position: absolute;
+  left: 0;
+  top: -0.4rem;
+  pointer-events: none;
+  transition: top 200ms;
+  font-size: 1rem;
+  color: var(--app-dark-gray);
+  opacity: 0.8;
+  font-weight: 400;
+}
+
+.auth-email > input:focus + label {
+  top: -1rem;
+  font-size: 12px;
+}
+.auth-email > input:focus {
+  border-bottom: 1px solid var(--primary);
+}
+.auth-email > a {
+  position: absolute;
+  bottom: 0.5rem;
+  right: 0;
+  font-size: 0.9rem;
+  color: var(--primary);
+  cursor: pointer;
+  font-weight: 900;
+}
+
+.terms {
+  text-align: left;
+  font-size: 0.7rem;
+  color: var(--app-light-gray);
+}
+.terms > a {
+  color: var(--primary);
+}
+.login-btn-container {
+  margin: 1rem 0.5rem 0.5rem 0.5rem;
+  width: 100%;
+}
+
+.login-actions {
+  margin-top: 4vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.register-link {
+  position: absolute;
+  bottom: 1rem;
+}
+.register-link {
+  color: var(--primary);
+  font-weight: 700;
+  cursor: pointer;
+}
+.or {
+  padding: 0.3rem;
+  color: var(--app-light-gray);
+  font-weight: 700;
+}
+
+.has-float-label {
+  display: block;
+  position: relative;
+  margin-bottom: 16px;
+}
+
+.has-float-label input:focus {
+  border-bottom: 2px solid #2979ff;
+  transition: all 0.2s;
+}
+
+.has-float-label input:disabled {
+  border-bottom: 2px dotted #8a8a8a !important;
```

Comment:
Don't use !important. it's not good practice.

Reply:
Yup noted will do that